### DISCUSSION TITLE
How to hide drafts

### DISCUSSION THREAD

**Discussion by JoelShepard** [adityatelange/hugo-PaperMod#643] [Q&A] [answered]
I tried to put 
```
buildDrafts: false
``` 
into my config.yml but it didn't work 
![image](https://user-images.githubusercontent.com/57812697/142988112-5022570a-b544-4efd-beee-91cd2ad19811.jpeg)

**Comment by danielfdickinson** [comment]
Hmmm.... Try also using: ``buildFuture: false``
Both configuration variables are like ``baseURL`` so don't go under ``params``.
HTH.

**Reply by JoelShepard** [reply]
Don't work.
![image](https://user-images.githubusercontent.com/57812697/143070246-3c66cf2f-10b2-47f9-af88-8c5d27450215.jpeg)
I don't know how to solve

**Comment by danielfdickinson** [comment]
Can you share your repo? What you've shown ought to work, so there must be something else in play.

**Comment by Mudhenn** [comment, marked as answer]
I came here because I had the same question and it took me a while to recognize that I still was starting with
`hugo server -D --disableFastRender `
out of habit.
Well, when I finally removed the D it behaved as expected. :blush:

**Reply by JoelShepard** [reply]
Thank you.
I solved simply removing -D into netlify deploy settings
